### DISCUSSION TITLE
Re ouverture de scène avec paramètre différent

### DISCUSSION THREAD

**Discussion by ArmandGC** [witsa/synapps#690] [Q&A]
Bonjour,
Je chercherais à Ouvrir une scène "Sous-station" via un bouton de mon menu (jusque là ça va), j'ai ajouter un additionnel à ma scène pour lui indiquer une sous-station a afficher
Voici la scène en question
![synapps-studio_IWpJ6C30UC](https://github.com/witsa/synapps/assets/134411061/5e3e9d29-c348-4fb5-9678-ccff31b03fb0)
Le composite au dessus à pour contexte la variable relative.

Sur mon gabarit j'ai donc le bouton qui va ouvrir cette sous-station et qui à donc en paramètre "Nod_1" qui correspond à ma première sous station.
![synapps-studio_aUl1burgSF](https://github.com/witsa/synapps/assets/134411061/2bf3ddef-1e67-46b0-af1c-3cef22950b3e)
Le bouton fonctionne bien,
Le problème étant que si j'ouvre la scène sous-station (Nod_1) et que j'essai ensuite d'ouvrir la même scène sous-station avec un second bouton qui à pour paramètre "Nod_2" rien ne ce passe.
![explorer_QyOBnvrad7](https://github.com/witsa/synapps/assets/134411061/44c07522-7291-4310-a7e4-3cb3862b6c92)
y'a t'il une solution pour que au moins le paramètre Nod_ ce mette à jour?
J'ai 24 sous-station cela me permettrait d'éviter d'avoir 24 scènes différents avec le même composite à l'intérieur.

**Comment by manufitoussiwit** [comment]
Salut! 
Il est fort probable que l'acteur écran n'a pas d'additionnelle Nod_ qui paramètre la scène Sous-Stat_Relatif_Tewt. 
Si vous aller sur l'acteur écran, sélectionnez l'affichage de cette scène juste le temps pour l'aider à fabriquer l'additionnelle Nod_.

En fait, l'acteur écran doit avoir la même clé d'additionnelle que le paramètre de la scène qu'elle affiche. 
Pour tout dire, normalement, ça ne devrait pas marcher non plus au premier affichage :-) On va étudier pour faire en sorte qu'il n'y ait plus cette contrainte.

**Reply by ArmandGC** [reply]
Salut,
Je pense que l'additionnel en l'occurrence n'a rien à voir avec le problème.
Même sans additionnel si j'ouvre une scène via un bouton et que j'essai de re ouvrir la même scène avec le même bouton (ou avec un autre) mon gabarit considère que la scène étant déjà ouverte dans l'acteur screen il n'a pas à la re ouvrir et donc il ne re charge pas ma page et ne peux donc pas mètre a jour mon additionnel.

**Reply by manufitoussiwit** [reply]
Oui c'est précisément ce qui se passe, et pour que la scène reçoive le paramètre il faut que l'acteur écran ait lui aussi cette additionnelle.

En réalité, il faut oublier un moment le bouton de navigation, en vrai, c'est l'acteur écran qui fait tout le boulot : il affiche la bonne scène et il lui passe les bons paramètres. Pour qu'il le fasse bien, il doit avoir les memes additionnelles que la scène à paramétrer.

Le bouton de navigation ne rien d'autre que changer (ou pas) la scène à montrer d'un acteur écran, et changer les additionnelles de l'acteur écran.

**Reply by ArmandGC** [reply]
Je viens effectivement de tester d'ajouter l'additionnel Nod_ à l'acteur Screen et cela semble bien fonctionner.
